packages/xhost.rb

```diff
@@ -0,0 +1,23 @@
+require 'package'
+
+class Xhost < Package
+	description 'Server access control program for X'
+	homepage 'https://github.com/freedesktop/xorg-xhost'
+	version '1.0.7'
+	source_url 'https://github.com/freedesktop/xorg-xhost/archive/xhost-1.0.7.tar.gz'
```

Comment:
Pleas change `source_url` to `https://www.x.org/releases/individual/app/xhost-1.0.7.tar.bz2`.

Reply:
Not a good idea to use a homepage with a version in it.  His homepage is fine.

Reply:
`X11R6.8.1` is just the version of the X server. No correlation to the package used. In fact, I believe that's the only version of the X server that has that manpage. It was split out into a separate package in later versions.